terraform/database.tf

```diff
@@ -255,6 +256,29 @@ resource "google_secret_manager_secret_version" "db-verification-code-hmac" {
   secret_data = join(",", reverse(random_id.db-verification-code-hmac.*.b64_std))
 }
 
+# Create secret for the database HMAC for phone numbers
+resource "random_id" "db-phone-number-hmac" {
+  count       = var.db_phone_number_hmac_count
+  byte_length = 128
```

Comment:
Some of the comments say 256?

Reply:
this is the HMAC secret length, not the nonce length